dataactvalidator/scripts/loadFSRS.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import func
+
+from dataactcore.models.baseInterface import databaseSession
+from dataactcore.models.fsrs import FSRSAward
+from dataactvalidator.fsrs import retrieveAwardsBatch
+
+def loadFSRS(minId):
+    """Fetch and store a batch of Award/Sub-Award records, replacing previous
+    entries."""
+    awards = list(retrieveAwardsBatch(minId))
+    ids = [a.id for a in awards]
+    with databaseSession() as sess:
```

Comment:
Eh, gonna go ahead and page @nmonga91 too so he can see the context manager at work and provide his .02.